bin/picca_compute_fvoigt_hcd.py

```diff
@@ -161,7 +113,8 @@ def main():
     wave = np.arange(2000, 8000, 1)  # TODO this grid may be too sparse
     integrand = np.empty((dN_NHI.size, wave.size // 2 + 1))
     for i, NHI in enumerate(dN_NHI):
-        profile_wave = get_voigt_profile_wave(wave, args.z_dla, NHI)
+        profile_wave = np.exp(
+            -compute_tau(wave, args.z_dla, NHI, LAMBDA_LYA, OSCILLATOR_STRENGTH_LYA, GAMMA_LYA)
```

Comment:
should the Lyβ absorption be added as well as we do this in the DLA mask? If not this does exactly the same thing as in the masking function, so it should be fine. Or rather, if it weren't fine we should not change things here, but over there...

Reply:
I'd be happy to add it, yes. What do you think @andreicuceu ?

Reply:
we would just need to replace line 116 by `profile_wave = dla_profile(wave, args.z_dla, NHI)` (and fix the imports)